src/axom/quest/detail/PointInCellMeshWrapper_mfem.hpp

```diff
@@ -432,6 +498,12 @@ class PointInCellMeshWrapper<quest_point_in_cell_mfem_tag>
 private:
   mfem::Mesh* m_mesh;
   bool m_isHighOrder;
+
+  // Parameters for inverse transformation
+  int m_printLevel {-1};
```

Comment:
Should `m_printLevel` be moved above this comment?

Reply:
It's actually also one of the parameters to `mfem::InverseTransformation`

Perhaps I should rename it to something like `m_queryPrintLevel` ? (Not sure if that's better though)